azurerm/internal/services/databasemigration/resource_arm_database_migration_project.go

```diff
@@ -0,0 +1,190 @@
+			TargetPlatform: datamigration.ProjectTargetPlatform(targetPlatform),
+		},
+		Tags: tags.Expand(t),
+	}
+
+	if _, err := client.CreateOrUpdate(ctx, parameters, resourceGroup, serviceName, name); err != nil {
+		return fmt.Errorf("Error creating Database Migration Project (Project Name %q / Service Name %q / Group Name %q): %+v", name, serviceName, resourceGroup, err)
+	}
+
+	resp, err := client.Get(ctx, resourceGroup, serviceName, name)
+	if err != nil {
+		return fmt.Errorf("Error retrieving Database Migration Project (Project Name %q / Service Name %q / Group Name %q): %+v", name, serviceName, resourceGroup, err)
+	}
+	if resp.ID == nil || *resp.ID == "" {
+		return fmt.Errorf("Cannot read Database Migration Project (Project Name %q / Service Name %q / Group Name %q) ID", name, serviceName, resourceGroup)
+	}
+	d.SetId(*resp.ID)
+
+	return resourceArmDatabaseMigrationProjectRead(d, meta)
+}
+
+func resourceArmDatabaseMigrationProjectRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).DatabaseMigration.ProjectsClient
+	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	id, err := azure.ParseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	name := id.Path["projects"]
+	resourceGroup := id.ResourceGroup
+	serviceName := id.Path["services"]
+
+	resp, err := client.Get(ctx, resourceGroup, serviceName, name)
+	if err != nil {
+		if utils.ResponseWasNotFound(resp.Response) {
+			log.Printf("[INFO] Database Migration Project %q does not exist - removing from state", d.Id())
+			d.SetId("")
+			return nil
+		}
+		return fmt.Errorf("Error reading Database Migration Project (Project Name %q / Service Name %q / Group Name %q): %+v", name, serviceName, resourceGroup, err)
+	}
+	if resp.ID == nil || *resp.ID == "" {
+		return fmt.Errorf("Cannot read Database Migration Project (Project Name %q / Service Name %q / Group Name %q) ID", name, serviceName, resourceGroup)
+	}
+
+	d.SetId(*resp.ID)
+
+	d.Set("name", resp.Name)
+	d.Set("service_name", serviceName)
+	d.Set("resource_group_name", resourceGroup)
+
+	location := ""
+	if resp.Location != nil {
+		location = azure.NormalizeLocation(*resp.Location)
+	}
+	d.Set("location", location)
+
+	if prop := resp.ProjectProperties; prop != nil {
+		d.Set("source_platform", string(prop.SourcePlatform))
+		d.Set("target_platform", string(prop.TargetPlatform))
+	}
+
+	return tags.FlattenAndSet(d, resp.Tags)
+}
+
+func resourceArmDatabaseMigrationProjectDelete(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*clients.Client).DatabaseMigration.ProjectsClient
+	ctx, cancel := timeouts.ForDelete(meta.(*clients.Client).StopContext, d)
+	defer cancel()
+
+	id, err := azure.ParseAzureResourceID(d.Id())
+	if err != nil {
+		return err
+	}
+	resourceGroup := id.ResourceGroup
+	serviceName := id.Path["services"]
+	name := id.Path["projects"]
+
+	deleteRunningTasks := true
```

Comment:
Could we switch this to false as a safe default. Now that we have the provider feature block we could add an option there if we so desire.

Reply:
I think we can put that into another PR, since it also impacts the service resource.